example_configs/weblogic.yml

```diff
@@ -0,0 +1,60 @@
+---
+jmxUrl: service:jmx:iiop:///jndi/iiop://localhost:7001/weblogic.management.mbeanservers.runtime
```

Comment:
We always recommend running as a JMX agent, so this should not be present.

Reply:
This is still here

Reply:
If I remove this line, I get no metrics at all.

Reply:
Why not? What's non-standard with this application?

Reply:
Well, I don't know exactly. Maybe there's something wrong with WebLogic in the way the agent retrieve informations.
Is there something I can do about that?

Reply:
Is it using something other than the default mbean server?

Reply:
Good idea. If I add the following parameters ```-Djavax.management.builder.initial=weblogic.management.jmx.mbeanserver.WLSMBeanServerBuilder``` to my weblogic server I get all the weblogic metrics.
Is there any option I can use to specify this class?

Reply:
You can pass it on that command line, I'd suggest putting that information in a comment up the top of the file.

Reply:
Fine, I update my PR with this and your previous comments.
Thanks for your feedback!